filebeat/module/nats/log/test/test.log-expected.json

```diff
@@ -60,7 +60,10 @@
         "input.type": "log",
         "log.level": "debug",
         "log.offset": 269,
-        "message": "172.18.0.1:38630 - cid:1 - Client connection created",
+        "nats.log.client.id": "1",
+        "nats.log.client.ip": "172.18.0.1",
```

Comment:
@webmat Perhaps you can chime in here. Looks to me like we could align this with ECS.

Reply:
Agreed. We should always try to populate `message` with the human readable part of the message.

It's fine to take out pieces that are metadata, like the header/preamble we see here. But the essence of the message should stick around in the message field: "Client connection created".

If you have a case where taking out all of the extracted bits removes meaning, it's fine to leave them in the message as well, e.g. if the message had been "Client connection accepted from 172.18.0.1:38630".

Reply:
Oh and nats.log.client.ip/port should definitely be mapped to `client.ip` and `client.port` as well, yeah

Reply:
changes are applied to next commits